Firmware/Marlin_main.cpp

```diff
@@ -7786,14 +7786,14 @@ SERIAL_PROTOCOLPGM("\n\n");
 
         updatePID();
         SERIAL_PROTOCOLRPGM(MSG_OK);
-        SERIAL_PROTOCOL(" p:");
+        SERIAL_PROTOCOLPGM(" p:");
```

Comment:
I think flash usage increases because strings in ram are optimized much more than strings in progmem. So there will be duplicates in progmem, while in ram these are deduplicated. Sadly, I couldn't find a fix for this besides using PSTR only once and using the pointer on multiple PROTOCOLRPGM functions